content/blog/2021-08-31-aug-community-gems.md

```diff
@@ -0,0 +1,154 @@
+---
+title: August '21 Community Gems
+date: 2021-08-31
+description: |
+  A roundup of technical Q&A's from the DVC community.
+  This month: separate DVC pipelines, working with CML,
+  object storage as a data source, and more.
+descriptionLong: |
+  A roundup of technical Q&A's from the DVC community.
+  This month: separate DVC pipelines, working with CML,
+  object storage as a data source, and more.
```

Comment:
object storage - is it still part of the Q/A this time?

Reply:
Ope! Just cleaned that up.